release-content/0.14/release-notes/12013_Upstreaming_bevy_color.md

```diff
@@ -1 +1,31 @@
-TODO
+Colors are a huge part of building a good game: UI, effects, shaders and more all need fully-feature, correct and convenient color tools.
+Bevy now supports a wide range of color spaces, each with their own type (e.g. `LinearRgba`, `Hsla` or `Oklaba`),
+and offers a wide range of fully documented operations on and conversions between them.
+
+Critically, color mixing / blending is now supported: perfect for procedurally generating color palettes and working with animations.
+
+```rust
+use bevy_color::{palettes::tailwind, Mix};
+
+// Each color space now corresponds to a specific type
+let red = Srgba::rgb(1., 0., 0.);
+
+// All non-standard color space conversions are done through the `LinearRgba` color space
+// to avoid a quadratic explosion of generated code.
+let red = Oklcha::from(LinearRgba::from(red));
```

Comment:
```suggestion
// All non-standard color space conversions are done through the shortest path between
// the source and target color spaces to avoid a quadratic explosion of generated code.
// This conversion...
let red = Oklcha::from(red);
// ...is implemented using
let red = Oklcha::from(Oklaba::from(LinearRgba::from(red)));
```

I want to show that a user doesn't need to worry about _how_ a color is actually converted, since we've already ensured every conversion pair is available and takes the shortest path using well known conversion algorithms.